services/apis/fence/fenceProps.js

```diff
@@ -34,7 +34,6 @@ module.exports = {
     registerGoogleServiceAccount: `${rootEndpoint}/google/service_accounts`,
     deleteGoogleServiceAccount: `${rootEndpoint}/google/service_accounts`,
     getGoogleServiceAccounts: `${rootEndpoint}/google/service_accounts`,
-    getGoogleServiceAccountMonitor: `${rootEndpoint}/google/service_accounts/monitor`,
```

Comment:
is this change necessary? we should still be able to use this for the monitoring SA

Reply:
I would rather not hard-code the SA if possible

Reply:
codeceptjs doesn't let us send requests during the test setup :(